src/fulltrade/FullTradeCard.js

```diff
@@ -67,7 +67,7 @@ export default class FullTradeCard extends Component {
                         <FullTradeParams
                             {...this.props}
                             disabled={disabled}
-                            contract={disabled ? mockedContract : contract}
```

Comment:
Nothing suspicious there 😉 . I don't think it should be there to begin with. { disabled }  is partly determined by server response , hence mockedContract is often passed regardless of the value of contract . Moreover , we performed similar check on contract above .